src/Concordium/Client/Output.hs

```diff
@@ -1074,7 +1074,7 @@ printChainParametersV0 ChainParameters {..} = tell [
   [i|     * fraction for the GAS account: #{_cpRewardParameters ^. tfdGASAccount}|],
   [i|  + GAS account distribution:|],
   [i|     * baking a block: #{_cpRewardParameters ^. gasBaker}|],
-  [i|     * adding a finalization proof: #{Types.uncond $ _cpRewardParameters ^. gasFinalizationProof}|],
+  [i|     * adding a finalization proof: |] <> showConditionally (_cpRewardParameters ^. gasFinalizationProof),
```

Comment:
It technically did not, but it seemed consistent to use it to print values wrapped in `Conditionally`. In case a V3 is added, it would hopefully be clear that this should be used to print such if someone decides to copy and piggyback off of this function in the future. In any case, should I revert it?

Reply:
No. I don't think there is a danger you are talking about here, but fine.